src/query/block/types.go

```diff
@@ -46,122 +48,125 @@ type Block interface {
 // UnconsolidatedBlock represents a group of unconsolidated series across a time bound
 type UnconsolidatedBlock interface {
 	io.Closer
-	// StepIter returns an UnconsolidatedStepIter
+	// StepIter returns a step-wise block iterator, giving unconsolidated values
```

Comment:
Thanks a lot for updating these to make them more informative :)

Reply:
+1